internal/pkg/csumio/csumreader.go

```diff
@@ -71,3 +71,7 @@ func (cr *Reader) Read(p []byte) (int, error) {
 	}
 	return n, nil
 }
+
+func GetChecksumFilePath(path string) string {
+	return path + ".csum"
+}
```

Comment:
I don't think it's appropriate to place this function in the csumio package because csumio doesn't depend on the file path suffix. The checksum file name should be decided the same way as `raw.img`, `fin.sqlite3`, etc., so how about moving this function to `internal/infrastructure/nlv`? And the file path of the checksum file will need to be passed to various functions, as `raw.img` does.